VideoStabilization/video_stabilization.py

```diff
@@ -91,7 +91,11 @@ def fixBorder(frame):
   curr_pts = curr_pts[idx]
 
   #Find transformation matrix
-  m = cv2.estimateRigidTransform(prev_pts, curr_pts, fullAffine=False) #will only work with OpenCV-3 or less
+  if float((cv2.__version__)[0:3])<=3.0:
```

Comment:
Based on other usages in the repo, I less brittle version of this is:

`if int((cv2.__version__).split('.')) <= 3:`

Reply:
@dustinfreeman what changes do you suggest here to merge this PR?